### DISCUSSION TITLE
Option to use a pin code/password to enable export

### DISCUSSION THREAD

**Discussion by SalitaNgDiyos** [rubiot/ibiblia#85] [Ideas]
When working with a team this would provide Admin rights to the module or pin code owner to make the decision to publish the projects so to avoid unauthorize reproduction of modules.

**Comment by rubiot** [comment]
I'm not sure about that. In order to be effective the project file would have to be encrypted. With some knowledge the user still would be able to get the data from the database, or export a patch and import into a new project.
